tests/system/action/assignment/test_update.py

```diff
@@ -71,6 +72,75 @@ def test_update_wrong_id(self) -> None:
         model = self.get_model("assignment/111")
         assert model.get("title") == "title_srtgb123"
 
+    def prepare_voting_phase_test(
+        self, number_of_speakers: int, phase: str = "search"
+    ) -> None:
+        self.create_meeting()
+        for i in range(number_of_speakers):
+            self.create_user(f"user{i+1}", [3])
+        ids = list(range(1, number_of_speakers + 1))
+        self.set_models(
+            {
+                "meeting/1": {
+                    "assignment_ids": [1],
+                    "list_of_speakers_ids": [1],
+                    "assignment_candidate_ids": ids,
+                    "assignment_poll_add_candidates_to_list_of_speakers": True,
+                },
+                "assignment/1": {
+                    "title": "assignment_with_candidates",
+                    "meeting_id": 1,
+                    "list_of_speakers_id": 1,
+                    "candidate_ids": ids,
+                    "phase": phase,
+                },
+                "list_of_speakers/1": {
+                    "content_object_id": "assignment/1",
+                    "meeting_id": 1,
+                },
+                **{
+                    f"assignment_candidate/{i}": {
+                        "meeting_id": 1,
+                        "assignment_id": 1,
+                        "meeting_user_id": i,
+                    }
+                    for i in ids
+                },
+                **{f"meeting_user/{i}": {"assignment_candidate_ids": [1]} for i in ids},
```

Comment:
I think `assignment_candidate_ids` should contain `i` to create a correct relation to the `meeting_user`.